dataactvalidator/scripts/loaderUtils.py

```diff
@@ -10,7 +10,7 @@ def clean_col_names(field):
 
 class LoaderUtils:
     @classmethod
-    def padFunction(self, field, padTo, keepNull):
+    def padFunction(cls, field, padTo, keepNull):
```

Comment:
I know CM already said it, but I'll reply here too. I planned to do a bunch of static method moving in the next round, I just wanted it to not get too big.